internal/plugin/connectors/tcp/mssql/connection_details.go

```diff
@@ -1,59 +1,62 @@
 package mssql
 
 import (
-	"bytes"
 	"strconv"
 )
 
 // ConnectionDetails stores the connection info to the real backend database.
 // These values are pulled from the SingleUseConnector credentials config
 type ConnectionDetails struct {
-	Host     string
-	Port     uint
-	Username string
-	Password string
-	SSLMode  string
+	Host       string
+	Port       uint
+	Username   string
+	Password   string
+	SSLMode    string
+	SSLOptions map[string]string
+}
+
+var sslOptions = []string{
+	"sslrootcert",
+	"sslkey",
+	"sslcert",
 }
 
 const defaultMSSQLPort = uint(1433)
+const defaultSSLMode = "disable"
```

Comment:
`defaultSSLMode` is only being used in the tests. I think on line 52 and 54, instead of the current string literal.